src/data/technologyBrands.json

```diff
@@ -0,0 +1,67 @@
+[
+  {
+    "name": "React Native",
+    "logo": "/img/react_native.jpg",
+    "web": ""
```

Comment:
If this is suppose to be the URL to the technology, change it to "route" to keep consistency with "navigation" file.